README.md

```diff
@@ -81,15 +81,50 @@ place to understand the nuances of Dotenv file parsing.
 You can call dotenv from the command line in order to load settings
 from one or more dotenv file before invoking an executable:
 
-```
-dotenv -f mydotenvfile myprogram
+```shell
+$ dotenv -e mydotenvfile myprogram
 ```
 
 Aditionally you can pass arguments and flags to the program passed to
 Dotenv:
 
+```shell
+$ dotenv -e mydotenvfile myprogram -- --myflag myargument
+```
+
+or:
+
+```shell
+$ dotenv -e mydotenvfile "myprogram --myflag myargument"
+```
+
+Also, you can use a `-x` to use `dotenv-safe` functionality so that you can have
```

Comment:
Can you please add a link to the npm dotenv-safe library so that people have a better idea of what we're talking about?